scopes/component/isolator/isolator.main.runtime.ts

```diff
@@ -737,6 +747,27 @@ export class IsolatorMain {
     return allCapsuleList;
   }
 
+  private async calcDependenciesGraph(
```

Comment:
this should be renamed as it's not only calc, it's doing a side affect - it's mutating the components.